### DISCUSSION TITLE
DEM error

### DISCUSSION THREAD

**Discussion by Enrique30065** [isce-framework/isce2#900] [General]
I have this error:
FileNotFoundError: [Errno 2] No such file or directory: 'Coarse_demLat_N37_N39_Lon_W119_W117.dem.wgs84.xml'
I cannot generate the geocode product. Do you know how to solve this error?

**Comment by Hassan22113** [comment]
Hi. Can you elaborate your question a bit. You're trying to geocode and you found this error or you are running the step to verifyDEM ?
